pkg/cmd/adm/unregister_member.go

```diff
@@ -62,5 +66,31 @@ func UnregisterMemberCluster(ctx *clicontext.CommandContext, clusterName string)
 	}
 	ctx.Printlnf("\nThe deletion of the Toolchain member cluster from the Host cluster has been triggered")
 
-	return restartHostOperator(ctx, hostClusterClient, hostClusterConfig.OperatorNamespace)
+	return restartHostOperator(hostClusterConfig)
+}
+
+func restartHostOperator(hostConfig configuration.ClusterConfig) error {
+	kubeConfigFlags := genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag()
+	hFactory := cmdutil.NewFactory(cmdutil.NewMatchVersionFlags(kubeConfigFlags))
+	ioStreams := genericclioptions.IOStreams{
+		In:     nil, // Not to forward the Standard Input
+		Out:    os.Stdout,
+		ErrOut: os.Stderr,
+	}
+
+	hArgs := []string{"", "deployments",
+		"--namespace", hostConfig.OperatorNamespace,
+		"--server", hostConfig.ServerAPI,
+		"--token", hostConfig.Token}
+
+	o := kubectlrollout.NewRolloutRestartOptions(ioStreams)
```

Comment:
this seem to be very similar to what we already have in `base_kubectl.go`, let's try to reuse the already existing  code if possible.
and as I mentioned in the other comment, it would be nice to wait.